packaging/greenboot/functions.sh

```diff
@@ -98,7 +98,8 @@ function wait_for() {
     shift 1
 
     local -r start=$(date +%s)
-    until ("$@"); do
+    until (timeout "${timeout}" bash -x -c $@); do    
```

Comment:
Since this code runs in production, I think we may want to remove `bash -x`. If the caller wants tracing, they will pass `bash -x` as part of the command.